.travis.yml

```diff
@@ -1,38 +1,28 @@
 language: python
 cache: pip
 services:
-- docker
+  - docker
 env:
   global:
-  - DOCKER_COMPOSE_VERSION: 1.25.5
-  - SERVICE_NAME: pisces-web
-  - APPLICATION_NAME: pisces
+    - CONTAINER: pisces-web
+    - APPLICATION_NAME: pisces
+    - DJANGO_PORT: 8007
 before_install:
-- sudo rm /usr/local/bin/docker-compose
-- curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname
-  -s`-`uname -m` > docker-compose
-- chmod +x docker-compose
-- sudo mv docker-compose /usr/local/bin
-- cp ${APPLICATION_NAME}/config.py.example ${APPLICATION_NAME}/config.py
-- echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-- docker-compose up -d
-- sleep 20s
-- docker-compose exec ${SERVICE_NAME} pip install coverage pre-commit
-- docker-compose exec ${SERVICE_NAME} pre-commit install
-install: true
-before_script:
-- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-0.7.0-linux-amd64
-  > ./cc-test-reporter
-- chmod +x ./cc-test-reporter
-- ./cc-test-reporter before-build
+  - cp ${APPLICATION_NAME}/config.py.example ${APPLICATION_NAME}/config.py
+  - echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
+  - docker-compose up -d
+install:
+  - ./wait-for-it.sh $CONTAINER:$DJANGO_PORT -- docker-compose exec $CONTAINER pip install coverage pre-commit
+  - docker-compose exec $CONTAINER pre-commit install
 script:
-- docker-compose exec ${SERVICE_NAME} pre-commit run --all-files --show-diff-on-failure
-- docker-compose exec ${SERVICE_NAME} coverage run manage.py test
-- docker-compose exec ${SERVICE_NAME} coverage xml
-- ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+  - docker-compose exec $CONTAINER pre-commit run --all-files --show-diff-on-failure
+  - docker-compose exec $CONTAINER coverage run manage.py test
+  - docker-compose exec $CONTAINER coverage xml
```

Comment:
You don't need this line. This just creates an XML version of the coverage report, which we used to ship off to Code Climate but don't anymore. https://coverage.readthedocs.io/en/coverage-5.5/cmd.html#cmd-xml